circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/CirceAkkaSerializer.scala

```diff
@@ -163,7 +165,7 @@ abstract class CirceAkkaSerializer[Ser <: AnyRef: ClassTag](system: ExtendedActo
    */
   protected def genericCodec: Codec[Ser] = Codec.from(genericDecoder, genericEncoder)
 
-  protected def genericEncoder: Encoder[Ser] =
+  private def genericEncoder: Encoder[Ser] =
```

Comment:
This visibility change is unrelated to the tuple=>case class change, right? 🤔

Reply:
Isn't it more related to #98 (?)

Reply:
It's a refactor I wanted to do and decided to do it now, because it is not backward compatible.

Reply:
Btw if you could open a PR to Hydra which adapts to all the new changes (probably as v0.3.3 then), after doing #98